src/app/(private)/roles/[id]/page.tsx

```diff
@@ -66,6 +66,7 @@ const Page: NextPage<{ params: { slug: string; id: string } }> = ({
   const [minPermissionError, setMinPermissionError] = useState<string | null>(
     null
   )
+  const [refreshData, setRefreshData] = useState<boolean>(false)
 
   const createUpdatedRoleData = (
```

Comment:
bu state sanırım sadece sayfaya refresh atmak için tutuluyor, refresh atmak için böyle bir state tutmaya gerek var mı? "refreshData" sadece useeffect dependency'sinde kullanılıyor?

Reply:
useEffect'i tetiklemesi için ekledim. !prev kısmı sadece tetiklensin diye kullandım.